packages/frontend/src/lib/BootcFolderColumn.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+import { bootcClient } from '../api/client';
+import { BootcBuildInfo } from '/@shared/src/models/bootc';
+
+export let object: BootcBuildInfo;
+
+function openFolder(folder: string) {
+  bootcClient.openFolder(folder);
+}
+</script>
+
+<button class="hover:cursor-pointer flex flex-col" on:click="{() => openFolder(object.folder)}">
```

Comment:
Should really use a Link. Either way, violet is one of the 'historic' colors that we aren't supposed to use.